src/components/LifeItems.svelte

```diff
@@ -0,0 +1,92 @@
+<script>
+  import { fade, fly } from 'svelte/transition';
+  import ExpandableSlots from './Expandable/ExpandableSlots.svelte';
+  import { birthDate } from '../stores/birthDate';
+  import { getLifeCount } from '../helpers/calculations';
+  import Preview from './Preview.svelte';
+  import CellsContainer from './CellsContainer/CellsContainer.svelte';
+  import { getCellSize, getCellProps } from './CellsContainer/helpers';
+  import arrow from '../../assets/images/arrow-black.svg';
+  import { toCapitalize } from '../helpers/common';
+
+  export let name = 'years';
+
+  const widthMap = {
+    weeks: 'max-w-6xl',
+    months: 'max-w-5xl',
+    years: 'max-w-2xl'
+  };
+
+  let liveCount, lifeItems, lived, total, size;
+
+  $: {
+    if ($birthDate) {
```

Comment:
looks like a hack )
I think observable should have some better way
However I need to investigate that first